### PR TITLE
fix: expandShorthands fails and prevent builder to load

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Cherry-picked here https://github.com/webstudio-is/webstudio/pull/3674

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: packages/css-data/src/shorthands.test.ts [hunk 1]
```diff
@@ -1448,3 +1448,7 @@ test("expand position-try", () => {
     ["position-try-options", "--dashed-ident"],
   ]);
 });
+
+test("Does not fail on empty transition", () => {
+  expect(expandShorthands([["transition", ""]])).toEqual([]);
```
Comment (TrySound) [inline packages/css-data/src/shorthands.test.ts:1453]: weird one

**Review by TrySound — APPROVED** [review]

File: packages/css-data/src/shorthands.test.ts (hunk 1 above)
Comment (kof) [reply packages/css-data/src/shorthands.test.ts:1453]: I think any value can be empty, we need to test it with empty string for all properties

File: packages/css-data/src/shorthands.test.ts [hunk 2]
```diff
@@ -1448,3 +1448,77 @@ test("expand position-try", () => {
     ["position-try-options", "--dashed-ident"],
   ]);
 });
+
+test("Does not fail on empty value", () => {
```
Comment (kof) [inline packages/css-data/src/shorthands.test.ts:1452]: is there any way we could get a full list of all properties  as an array and get a loop over it, then add a bunch of values beyond empty string to get generated tests for all of them?

Comment (kof) [reply packages/css-data/src/shorthands.test.ts:1452]: I don't think its feasible to maintan a list of properties here
